tasks/setup.py

```diff
@@ -28,13 +28,16 @@ def setup(_: Context, config_path: Optional[str] = None, copy_to_clipboard: Opti
     else:
         os.system("brew install pulumi/tap/pulumi")
 
+    # install plugins
+    os.system("pulumi --non-interactive plugin install")
```

Comment:
Note that it needs to be run with CWD == `test-infra-definitions` repo.

Reply:
Worth adding a cwd here, anyhow as the script is defined in `test-infra`, it is mostly called from there. But I know of errors triggered because they called it from somewhere else

Reply:
Also realising we did not replace os.system with `ctx.run` here, on it

Reply:
What am I saying, I should run from the `tasks` parent !